adagrams/game.py

```diff
@@ -1,11 +1,113 @@
+import random
+
+LETTER_POOL = {
+    'A': 9, 
+    'B': 2, 
+    'C': 2, 
+    'D': 4, 
+    'E': 12, 
+    'F': 2, 
+    'G': 3, 
+    'H': 2, 
+    'I': 9, 
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+}
+
+LETTER_SCORES = {
+    'A': 1, 
+    'B': 3, 
+    'C': 3, 
+    'D': 2, 
+    'E': 1, 
+    'F': 4, 
+    'G': 2, 
+    'H': 4, 
+    'I': 1, 
+    'J': 8, 
+    'K': 5, 
+    'L': 1, 
+    'M': 3, 
+    'N': 1, 
+    'O': 1, 
+    'P': 3, 
+    'Q': 10, 
+    'R': 1, 
+    'S': 1, 
+    'T': 1, 
+    'U': 1, 
+    'V': 4, 
+    'W': 4, 
+    'X': 8, 
+    'Y': 4, 
+    'Z': 10
+}
+
+
 def draw_letters():
-    pass
+
+    letter_pool = LETTER_POOL.copy()
+    letters = []
+    for i in range(10):
+        letter = random.choice(list(letter_pool.keys()))
+        while letter_pool[letter] == 0:
+            letter = random.choice(list(letter_pool.keys()))
+        letters.append(letter)
+        letter_pool[letter] -= 1
+
+    return letters
```

Comment:
I love the way you handled the variable number of each letter we have using the while loop to see if we have letters left! This is an interesting case because instead of copying everything over into one long list, you really make use of the dictionary structure, so really well done!